### PR TITLE
Add PatientInfo.js

### PR DESCRIPTION
**Pull request by rodonguyen**

### CODE REVIEW

File: FHIR-React/src/pages/PatientInfo.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    birthDate: patientData.birthDate,
+    communication: communication,
+  }
+  return data
+}
+
+
+const PatientInfo = () => {
+  const { id } = useParams()
+  const [patientData, setPatientData] = useState()
```
Comment (maxgoodwin) [inline FHIR-React/src/pages/PatientInfo.js]: should be `useState({})` instead of `useState()`, to initialise patientData as an object.

**Comment by maxgoodwin** [comment]
./FHIR-React/bin should be added to .gitignore

**Comment by maxgoodwin** [comment]
> ./FHIR-React/bin should be added to .gitignore

Should have clarified, the `.gitignore` inside the root project folder needs to be updated (i.e. the parent folder of `FHIR-React`. I should have deleted the .gitignore inside `FHIR-React` when I merged the two FHIR projects together.

**Review by maxgoodwin — APPROVED** [review]

**Comment by rodonguyen** [comment]
Closed to open newer & complete PR.

**Comment by rodonguyen** [comment]
Closed to open newer & complete PR.

**Pull request closed without merging**